tests/README.adoc

```diff
@@ -0,0 +1,68 @@
+= Test Runner
+
+This project contains a simple test runner for Protractor Angular tests. The intention is to have an 
+isolated test runner that does not share dependencies with code under test. This test runner works
+standalone and just needs to be pointed at a set of Typescript test specs and an URL to test them on.
+
+Please note that this test runner is for **black box tests** only!
+
+== Preparing the Test Runner
+
+The test runner installs it's own dependencies automatically. You just need `node/npm` available in
+the current context. You will also need to have `curl` and `bash` installed on your system 
+and available in the path. **You don't need to do `npm install` in the runner directory**.
+
+== Providing Test Specs
+
+To run a test suite, you need to create a set of test specs (which may include page objects). The
+test specs source directory needs to include a `tsconfig.json` configuration file. The path to the 
+spec source directory has to be provided using the env variable `TEST_SOURCE_PATH`:
+
+== Running Tests
+
+Next, you need to provide a URL of the system under test using the env variable `BASE_URL`. If 
+`BASE_URL` is not set, `http://localhost:8088/` is used.
+
+Tests can then be run with:
+
+```
+npm test
+```
```

Comment:
The readme needs to reflect how we do this in planner
1. Run planner (in runtime)
2. cd to tests folder
3. npm test

Reply:
Yeah, added a paragraph there. The idea was that /tests is more like independent and not necessarily only used with the Planner.